### PR TITLE
Added Tests and packaging for System.Windows.Extensions.

### PR DESCRIPTION
**Pull request by Anipik**
Fixes https://github.com/dotnet/corefx/issues/31203

- Packaging for System.Windows.Extensions.
- Implemenation was broken  due to handle definations. Copied the handle implementation from System.Security.X509Certificates namespace to fix it
- Manual UI tests are added with infra similar to how we do testing  for System.Console

### CODE REVIEW

File: src/System.Windows.Extensions/tests/X509Certificate2UIManualTests.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        internal string CurveEncodedOidHex { get; private set; }
+        internal string Name { get; private set; }
+
+        internal static readonly EccTestData Secp256r1Data = new EccTestData
+        {
+            Name = nameof(Secp256r1Data),
+            CurveOid = Secp256r1OidValue,
+            CurveEncodedOidHex = Secp256r1OidHexValue,
+
+            // Suite B Implementers Guide to FIPS 186-3,
```
Comment (danmosemsft) [inline src/System.Windows.Extensions/tests/X509Certificate2UIManualTests.cs]: Nit, you have a non-ASCII character here

File: src/System.Windows.Extensions/tests/X509Certificate2UIManualTests.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        public static void SelectCertificateSingleSelectionTest(int count)
+        {
+            using (ECDsa ecdsa = ECDsa.Create(EccTestData.Secp256r1Data.KeyParameters))
+            {
+                CertificateRequest request = new CertificateRequest("CN=Test", ecdsa, HashAlgorithmName.SHA256);
+                DateTimeOffset now = DateTimeOffset.UtcNow;
+
+                using (X509Certificate2 cert = request.Create(request.SubjectName, X509SignatureGenerator.CreateForECDsa(ecdsa), now, now.AddMinutes(10), new byte[1]))
+                {
+                    X509Certificate2Collection collection = new X509Certificate2Collection() { cert, cert };
```
Comment (danmosemsft) [inline src/System.Windows.Extensions/tests/X509Certificate2UIManualTests.cs]: Should there be comments in the test describing what I'm supposed to do manually?

File: pkg/descriptions.json [hunk 3]
```diff
@@ -2132,6 +2132,14 @@
       "System.Web.HttpUtility"
     ]
   },
+  {
+    "Name": "System.Windows.Extensions",
+    "Description": "Provides the functionality to display the X509Certificate2",
```
Comment (danmosemsft) [inline pkg/descriptions.json]: I assume we have grander plans for a package named "System.Windows.Extensions" than "display the X509Certificate2"

Maybe something like "Provides miscellaneous Windows-specific types" at least

File: src/System.Windows.Extensions/tests/X509Certificate2UIManualTests.cs (hunk 2 above)
Comment (Anipik) [reply src/System.Windows.Extensions/tests/X509Certificate2UIManualTests.cs]: Actually I added that as the title of the pop-up box
```
collection, $"Choose {count} Certificate", string.Empty,
```

**Comment by Anipik** [comment]
@dotnet-bot test Linux arm Release Build

**Comment by Anipik** [comment]
@dotnet-bot test  Linux arm Release Build

**Pull request merged**